MyOwnORM/MyOwnORM/Repository/CustomDbSetRepository.cs

```diff
@@ -0,0 +1,520 @@
+﻿using MyOwnORM.Reflection;
+using MyOwnORM.Repository;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class CustomDbSetRepository<T> : ICustomDbSetRepository<T> where T : class
```

Comment:
Since you moved ICustomDbSetRepository to appropriate folder then move this class to 'Implementations'

Reply:
Fixed